.vscode/tasks.json

```diff
@@ -16,4 +16,4 @@
             "taskName": "test"
         }
     ]
-} 
\ No newline at end of file
+}
```

Comment:
Nit: since I struggled over it

It is often considered _good practice_ to have a newline at end of a file.
Historically, it caused some issues not doing so.

Reply:
Thank you but the file is not among the changes anymore. 795417e should have reverted it. Right?

Reply:
As said, I do not feel strong about 👍 
Either way is fine